src/components/page/styles.module.css

```diff
@@ -140,7 +146,7 @@
 
 @media screen and (min-width: 1024px) {
     .pageContent {
-        overflow: hidden;
+        overflow-y: scroll;
```

Comment:
we are preventing overflow in web-view, aren't we?

Reply:
web view  mei jo dropdown hai vo show nhi hota completely isiliye lgaya hai vo

Reply:
768px pe bhi nhi show hora to vo height increase karde thodi, width pe lagane se kya hoga?